tasklist/Makefile

```diff
@@ -11,6 +11,23 @@ WITH_MULTI_TENANCY:=false
 # Bash script to write or replace a variable in a file with the format VAR=value, args: variable name, variable value, target file
 WRITE_OR_REPLACE_SCRIPT:=./config/write_or_replace_var.sh
 
+.PHONY: env-h2-up
+env-h2-up:
+	CAMUNDA_SECURITY_INITIALIZATION_USERS_0_USERNAME=demo \
+	CAMUNDA_SECURITY_INITIALIZATION_USERS_0_PASSWORD=demo \
+	CAMUNDA_SECURITY_INITIALIZATION_USERS_0_NAME=Demo \
+	CAMUNDA_SECURITY_INITIALIZATION_USERS_0_EMAIL=demo@example.com \
+	CAMUNDA_SECURITY_INITIALIZATION_DEFAULTROLES_ADMIN_USERS_0=demo \
+	CAMUNDA_SECURITY_AUTHENTICATION_UNPROTECTEDAPI=true \
+	CAMUNDA_SECURITY_AUTHORIZATIONS_ENABLED=false \
+	CAMUNDA_BACKUP_WEBAPPS_ENABLED=false \
+	CAMUNDA_DATABASE_TYPE=rdbms \
```

Comment:
there is no problem with this, but since the property is deprecated, it will print a warning saying so. No big deal